tests/test_local_json.py

```diff
@@ -135,6 +135,19 @@ def test_graph():
     assert s1.pred == 'hasRelatedSynonym'
     assert s1.xrefs == ['GOC:mah']
 
+    GOSLIM = 'goslim_generic'
+    subsets = ont.subsets(NUCLEUS)
+    print("SUBSETS: {}".format(subsets))
+    assert GOSLIM in subsets
```

Comment:
are we not using the unittest framework? This would be much easier to test with unittest and nose (which easily runs tests).